packages/cdk-routing-split/src/add/handler.ts

```diff
@@ -0,0 +1,25 @@
+import { APIGatewayEvent, Context } from 'aws-lambda';
+import { createLogger, Logger, serializeError } from '@daysmart/aws-lambda-logger';
+import { action } from './action';
+import { HttpError } from '../http-error';
+
+export const handler = async (event: APIGatewayEvent, context: Context): Promise<any> => {
+    let logger!: Logger;
+
+    try {
+        logger = createLogger(process.env.DEBUG === 'true', context.awsRequestId);
```

Comment:
It is convenient that logger can be activated by setting debug property to true inside lambda event. Can you make same condition for logger activation inside add, delete and update lambda like it is done for get-origin lambda? -> createLogger(!!(event?.debug || process.env.DEBUG), context.awsRequestId);
If you think this is not needed align condition for logger creation inside get-origin lambda with other lambdas.

Reply:
yea i missed this one thanks